src/core/pluralize.js

```diff
@@ -0,0 +1,68 @@
+// Adds automatic pluralization to dfns
+// If a dfn is referenced as it's plural (and plural of `data-lt` attributes),
+//   plurals of it are automatically added to `data-plurals`.
+// The linking is done in core/link-to-dfn
+
+import { plural as pluralOf } from "deps/pluralize";
+import { norm as normalize } from "core/utils";
+
+export const name = "core/pluralize";
+
+export function run(conf) {
+  if (!conf.pluralize) return;
+
+  if (!conf.hasOwnProperty("definitionMap")) {
+    conf.definitionMap = Object.create(null);
+  }
+  const pluralizeDfn = pluralizer();
+
+  document
+    .querySelectorAll("dfn:not([data-lt-no-plural]):not([data-lt-noDefault])")
+    .forEach(dfn => {
+      const terms = [dfn.textContent];
+      if (dfn.dataset.lt) terms.push(...dfn.dataset.lt.split("|"));
+
+      const plurals = Array.from(
+        terms
+          .map(pluralizeDfn)
+          .filter(plural => plural)
+          .reduce((plurals, plural) => plurals.add(plural), new Set())
+      );
+
+      if (plurals.length) {
+        dfn.dataset.plurals = plurals.join("|");
+        plurals.reduce((defMap, plural) => {
+          if (!defMap[plural]) defMap[plural] = [];
+          defMap[plural].push($(dfn));
+          return defMap;
+        }, conf.definitionMap);
+      }
+    });
+}
+
+function pluralizer() {
```

Comment:
pluralizer is a noun... can we make this a verb?

Reply:
`pluralizer` returns a function which causes pluralization. So, `pluralizer` isn't actually doing the pluralization itself.

We can rename it to `getPluralizer` maybe?